src/components/Layout/Navbar/index.jsx

```diff
@@ -0,0 +1,115 @@
+import { ACTIONS } from '../../../customHooks/Actions.types';
+import LoginForm from '../LoginForm';
+
+const Card = styled.div`
+height: 400px;
+`
+
+// eslint-disable-next-line react/prop-types
+function Navbar({ ...props }) {
+  const inputRef = useRef();
+  const passRef = useRef();
+  const userRef = useRef();
+  const toogleRef = useRef();
+  const history = useHistory();
+  const { data, setData } = useData();
+
+  const [show, setShow] = useState(false);
+
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+
+  const searchByTerm = () => {
+    setData({ type: ACTIONS.SET_SEARCH_TERM, payload: inputRef.current.value})
+    history.push('/');
+  }
+
+  const changeTheme = () => {
+    setData({ type: ACTIONS.CHANGE_THEME, payload: !data.isDarkThemeActive })
+  }
+
+  const login = () => {
+    loginApi(userRef.current.value, passRef.current.value)
+      .then(resp => {
+        setData({ type: ACTIONS.SET_SESSION_DATA, payload: resp });
+        localStorage.removeItem('session');
+        localStorage.setItem('session', JSON.stringify(resp));
+        NotificationManager.success('Ingresaste con éxito', 'Bienvenido', 3000);
+        handleClose();
+      })
+      .catch(error => {
+        NotificationManager.warning('Usuario o contraseña incorrectos', 'Ups', 3000);
+      })
+  }
+
+  const logout = () => {
+    setData({ type: ACTIONS.SET_SESSION_DATA, payload: null });
+    localStorage.removeItem('session');
+    NotificationManager.success('Sessión cerrada con éxito', 'Bye', 3000);
+  }
+
+  return (
+    <nav className={`navbar navbar-expand-lg navbar-${data.isDarkThemeActive ? 'dark' : 'light'} bg-${data.isDarkThemeActive ? 'dark' : 'light'}`}>
+      <div className="collapse navbar-collapse" id="navbarSupportedContent">
+        <div className="form-inline mr-auto my-lg-0">
+          <input role={'SearchInput'} className="form-control mr-sm-2" type="search" placeholder="Buscar" aria-label="Buscar" ref={inputRef} />
+          <button role={'SearchButton'} className="btn btn-outline-success my-2 my-sm-0" type="button" onClick={searchByTerm}>Buscar</button>
+        </div>
+        <ul className="navbar-nav">
+          <div className='mt-1 mx-2'>
+            <BootstrapSwitchButton role={'ThemeToggle'} size='sm' value={data.isDarkThemeActive} ref={toogleRef} onChange={changeTheme} />
+            <label className="custom-control-label mx-2">DarkMode</label>
+          </div>
+          <li className="nav-item dropdown">
+            <a className="nav-link dropdown" href="/" role={'UserButton'} id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+              <FontAwesomeIcon icon={faUser} />
+            </a>
+            <div className="dropdown-menu" aria-labelledby="navbarDropdown">
+              {
+                !data.sessionData &&
+                <a className="dropdown-item" onClick={handleShow}>Iniciar sesión</a>
```

Comment:
We could have this inline like 

```suggestion
                <a className="dropdown-item" onClick={() => setShow(false)}>Iniciar sesión</a>
```